packages/block-editor/src/components/block-switcher/pattern-transformations-menu.js

```diff
@@ -106,7 +110,6 @@ function BlockPattern( { pattern, onSelect, composite } ) {
 			<CompositeItem
 				role="option"
 				as="div"
```

Comment:
I can see a console warning related to the `as` prop being deprecated, we should use the `render` prop instead.